HW.06.TASK2/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace HW._06.TASK2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string make = null;
+            string model = null;
+            int miles = 0;
+            int power = 0;
+            int vol = 0;
+            string type = null;
+            Guid vinnumber = default;
+            
+            
+            Motorcycle motorcycle = new Motorcycle();
```

Comment:
you created an object but this one has only default values due to the fact all Properties are private